understanding/20/no-keyboard-trap.html

```diff
@@ -41,7 +41,7 @@ <h2>Benefits of No Keyboard Trap</h2>
       
       <ul>
          
-         <li>People who rely on a keyboard or keyboard interface to use the Web including people
+         <li>People who rely on a keyboard or keyboard interface to use the web including people
```

Comment:
This feels like it verges on being more like proper noun usage? 
> lowercase except if proper noun meaning World Wide Web

I can go either way; just flagging